Sources/NIOCore/Utilities.swift

```diff
@@ -93,7 +93,9 @@ public enum System {
             .map { $0.ProcessorMask.nonzeroBitCount }
             .reduce(0, +)
 #elseif os(Linux) || os(Android)
-        if let quota = Linux.coreCount(quota: Linux.cfsQuotaPath, period: Linux.cfsPeriodPath) {
+        if let quota2 = Linux.coreCount(maxPath: Linux.cfsCpuMaxPath) {
```

Comment:
Should we adapt the documentation of `coreCount` to also include details of how c groups are used?

Reply:
I've added a brief bit of extra commentary.  There is scope for broader review here.  The idea of using this number to setup your threads loop group is mentioned - in most cases this is probably the wrong thing to do.

I also wonder if the logic could be improved - I think it's probably possible to be both in a cgroup and a cpuset.  I do wonder if CPUSet combined with multiple threads is a good idea or not - suspect anyone doing this has probably put in quite a lot of thought.